### DISCUSSION TITLE
remove twitch channel points and the buy me coffee part of the webpage

### DISCUSSION THREAD

**Discussion by smackincheeks** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#710] [Q&A]
is there a way to remove the top portion of the webpage above the graph that says buy me coffee and the twitch point miner?

**Comment by XaaRii** [comment]
it's a html file in your assets folder, i think.
As long as you understand html, you can do whatever you want.

**Comment by rdavydov** [comment]
In my fork, I've added a checkbox that toggles that header in the Analytics UI.

https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2
